components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/cmd/BuildCmd.java

```diff
@@ -378,26 +363,6 @@ private void invokeCallHome() {
         }
     }
 
-    /**
-     * init configuration.
-     *
-     * @param configPath path of the configureation file.
-     */
-    private static void initConfig(String configPath) {
-        try {
-            Path configurationFile = Paths.get(configPath);
-            if (Files.exists(configurationFile)) {
-                Config config = TOMLConfigParser.parse(configPath, Config.class);
-                CmdUtils.setConfig(config);
-            } else {
-                logger.error("Configuration: {} Not found.", configPath);
-                throw new CLIInternalException("Error occurred while loading configurations.");
-            }
-        } catch (ConfigParserException e) {
-            logger.error("Error occurred while parsing the configurations {}", configPath, e);
-            throw new CLIInternalException("Error occurred while loading configurations.");
-        }
-    }
 
```

Comment:
One of the newlines can be removed.

Reply:
Fixed in 76b551e116b6a9964bddf801be0f597424a488be